py/tests/integration/test_spend_tx.py

```diff
@@ -6,14 +6,21 @@
 import time
 from nose.tools import assert_equals, assert_not_equals, with_setup
 import common
+import json
 from waiting import wait
 from swagger_client.models.ping import Ping 
+from swagger_client.models.tx import Tx
 from swagger_client.models.spend_tx import SpendTx
+from swagger_client.models.name_preclaim_tx import NamePreclaimTx
+from swagger_client.models.name_claim_tx import NameClaimTx
+from swagger_client.models.name_update_tx import NameUpdateTx
 from swagger_client.rest import ApiException
 
+import keys
+
 settings = common.test_settings(__name__.split(".")[-1])
 
-def test_successful():
+def _test_successful():
```

Comment:
What is impact of prefixing _ to name? Why?

Reply:
Ah, typo